detail.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Book Detail</title>
+  <meta name="description" content="training, book detail card">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <div class="container detail">
+    <div class="card">
+      <div class="cover">
+        <img class="cover__image" src="assets/book-cover.png" alt="Book cover">
+        <div class="cover__badge">
+          <img class="cover__badge__image" src="assets/badge.png" alt="Cover badge">
```

Comment:
Avoid to use HTML elements for decorative elements. You can use ::after or ::before pseudoelements.

Reply:
Great!